e2e/tests/integrations/integrator/create-integration.test.ts

```diff
@@ -2,12 +2,17 @@ import { test, expect } from "@playwright/test";
 import { createIntegrationAsIntegrator } from "./create-integration.js";
 import { IntegrationType } from "@app-types/IntegrationType";
 
-test.use({ storageState: "playwright/.auth/user.json" });
+const integrationTypeValues = Object.entries(IntegrationType);
 
-test("As an integrator I can create a new integration", async ({ page }) => {
-  const { integrationName } = await createIntegrationAsIntegrator(
+integrationTypeValues.map(([integrationName, integrationType]) => {
```

Comment:
I've discussed this once with @simon-debruijn and it's better to use `forEach` here instead of `map`.
 
 `map` is better to use if you want to modify something on the array and use the modified array.